src/api/apiInstance.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-param-reassign */
+import axios from 'axios';
+import { getTokenSecureStorage } from 'helpers';
+import baseURL from '../config';
+
+export const apiInstance = axios.create({
+  baseURL,
+});
+
+export const authInterceptor = () => {
+  apiInstance.interceptors.request.use(
```

Comment:
but yes not sure if it is necessary to split this logic into 2 files?

Reply:
yes im not sure whats the best way so i wanted to ask, because i thought it looks cleaner to have api setup and apis separately. do you think i should move this back together in one file?

Reply:
ii think it is ok either way, just apInstance should be exported from one file so one source of truth and the api client can got back to default export 

and tests on authInterceptor should pass anyway (i mean without separate function for this)